src/pages/patterns/drag-and-drop.mdx

```diff
@@ -0,0 +1,629 @@
+---
+
+![List of items with one item being dragged to show parts of the item.](figma:patterns-dnd-intro.png)
+
+## Anatomy
+
+[Drag and drop](https://zendeskgarden.github.io/react-components/?path=/story/packages-drag-drop-patterns--drag-and-drop)
+interaction is common in workflows where the user wants to change the order or of items.
+It is also often used to build something by moving items around the screen,
+for example, a layout, report, or dashboard.
+
+![List of items with one item being dragged to show parts of the item.](figma:patterns-dnd-anatomy.png)
+
+1. Draggable grip
+2. Target placeholder as whitespace
+3. Draggable item being dragged
+
+Drag and drop between lists puts more emphasis on dropzones and source placeholders.
+
+![A List containing four draggable items where one has a hover state, with a dropzone on the right side.](figma:patterns-dnd-anatomy-flow-1.png)
+![A list containing four draggable items where one is being dragged toward an active dropzone on the right side.](figma:patterns-dnd-anatomy-flow-2.png)
+![A list with four draggable items where one is being dragged over a highlighted dropzone on the right side.](figma:patterns-dnd-anatomy-flow-3.png)
+![A list containing three draggable items on the left side and one draggable item on the right side.](figma:patterns-dnd-anatomy-flow-4.png)
+
+1. Draggable item
+2. Dropzone
+3. Source placeholder
+4. Dropzone active
+5. Dropzone highlighted
+6. Draggable item released
+
+### Related components
+
+- [Draggable](/components/draggable)
+
+## Formatting and States
+
+### Draggable item
+
+The grip handle and border define the default [Draggable](/components/draggable) item. Draggable items should have labels
+that are short and easily identifiable. Sometimes these are user-generated so the length may vary.
+The grip icon provides a visual affordance to communicate drag-and-drop. Border communicates
+the boundaries of a draggable item. Draggable items with many interactive elements can restrict
+movement to the grip handle only.
+
+![Draggable item with default, grabbed, and disabled state comparison.](figma:patterns-dnd-formatting-draggable.png)
+
+1. **Default** state communicates what is draggable
+2. **Grabbed** state communicates the item moving
+3. **Disabled** state communicates a non-interactive item
+
+#### Dragging larger items
+
+Items can include other interactive elements inside them, such as buttons, anchors, or even form elements.
+Nested elements will make the parent element taller and harder to interact with. Instead of making the item transparent,
+transform it into a simpler draggable item while dragging.
+
+<BestPractice>
+  <Do
+    imageSource={
+      props.data.dndFormattingDraggablePreviewDo.childFile.childImageSharp.gatsbyImageData
+    }
+  >
+    <ul>
+      <li>Provide a simpler preview of a draggable item while dragging</li>
+    </ul>
+  </Do>
+  <Dont
+    imageSource={
```

Comment:
nit: this is a known behavior of the do/don't component, but these images cut off. probably have to adjust the dimensions of the frames manually in figma.

<img width="745" alt="image" src="https://user-images.githubusercontent.com/39511661/233448068-e15c012b-0276-477b-ad97-3f642b210239.png">